packages/gatsby-recipes/src/providers/npm/package.js

```diff
@@ -151,10 +152,17 @@ const validate = resource =>
 exports.validate = validate
 
 const destroy = async ({ root }, resource) => {
-  await execa(`yarn`, [`remove`, resource.name], {
+  const readResource = await read({ root }, resource.id)
+
+  if (!readResource) {
+    return undefined
+  }
+
+  await execa(`yarn`, [`remove`, resource.name, `-W`], {
```

Comment:
we should add a TODO here to add support for NPM too sooner than later

Reply:
Went ahead and made a full-fledged issue: #24567